pkg/controller/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package controller originate from k8s.io/kubernetes/pkg/controller
```

Comment:
We may need a more detailed doc to explain why we use this controller rather than the default one.

Reply:
there's no default one